pegasus/routes/v2_section_routes.rb

```diff
@@ -79,6 +79,7 @@
   JSON.pretty_generate(section.to_owner_hash[:students])
 end
 
+# DEPRECATED: Use POST /dashboardapi/sections/<id>/students/bulk_add
 post '/v2/sections/:id/students' do |id|
```

Comment:
@caleybrock Do you know if we can tear this API out yet?  If we can, there's one less bit of logic we have to convert for multi-auth. (cc @madelynkasula @ewjordan)

Reply:
this endpoint should be fully deprecated and unused. there's a [honeybadger notification](https://app.honeybadger.io/projects/34365/faults/38119800#notice-summary) logging when it is hit. it looks like the notification happened 5 times on june 6th and hasn't happened since, so i feel pretty confident that this can be removed.

Reply:
Hmm... that Honeybadger notification is for `GET /dashboardapi/sections/<id>/students` and this is `POST /dashboardapi/sections/<id>/students`.  Maybe I'll add one first and we'll get a day or so of data before we tear this out.